src/gui/Window.cpp

```diff
@@ -873,6 +873,14 @@ void Window::addFile(bool showOptions, const QString& startDir, const QString& f
 	this->markModified(true);
 }
 
+
+void Window::addFiles(bool showOptions, const QString &startDir) {
+	// Add multiple files using the multiple file selector
+	QStringList files = QFileDialog::getOpenFileNames(this, tr( "Open Files" ));
```

Comment:
extra space in the tr :P

speaking of tr, should the translations in `src/gui/EntryContextMenuData.h` be updated to say `Add Files` plural?

oh, and should `Edit > Add File...` be updated to use this method as well?

Reply:
This PR does update `Edit > Add File...`, it's higher up in the same file.

Will fix the space and text!

Reply:
fixed! also updated the other instances to be plural

Reply:
> This PR does update `Edit > Add File...`, it's higher up in the same file.

im tired

Reply:
No worries!